sa/database.go

```diff
@@ -98,6 +81,10 @@ func recombineURLForDB(dbConnect string) (string, error) {
 
 	dsnVals.Set("parseTime", "true")
 
+	// Required to make UPDATE return the number of rows matched,
+	// instead of the number of rows changed by the UPDATE.
+	dsnVals.Set("clientFoundRows", "true")
```

Comment:
Interesting. What's the story here? Is this just to make the MySQL driver's behavior the same as default behavior for sqlite?